utils/deployer/migrations/1_initial_migration.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config();
+var assert = require('assert');
+const Types = artifacts.require('Types');
+const Swap = artifacts.require('Swap');
+const Wrapper = artifacts.require('Wrapper');
+const Indexer = artifacts.require('Indexer');
+const DelegateFactory = artifacts.require('DelegateFactory');
+
+module.exports = async(deployer, network) => {
+
+  assert.notEqual(network, 'development', "Please choose a network besides 'development'")
```

Comment:
why not allow development? in that I think contract code etherscan verification should only be there for Rinkeby and Mainnet, but having a singular deploy script allows someone to test on a private network.